Works on redmine 2.3.2 stable, ruby 2, rails 3.2.13

cforce opened the pull request:
knows bugs

Clicking Nex and page size for requriements list still renders new page
todo: adapt js for this bindings too

kawasima committed:
#127 change labelled_tabular_form_for to labelled_form_for.

halimali committed:
initial method to display please create a version for IE

halimali committed:
fix NG testcase create issue error with custom field.
just set tracker id to 1 which is bug tracker

halimali committed:
add the watchers controls/elements to the NG issue dialog

halimali committed:
fix NG issue dialog, optional description is truncated

halimali committed:
flash error for test case not selected when press copy
and redirect to test case index

halimali committed:
Revert "initial method to display please create a version for IE"
This cause the error to be display if there is only 1 version

This reverts commit 803dcce195d8f0dd5fc3619604faa441e7c5c185.

halimali committed:
use params[:test_plan][:version_id] to check whether version already created
for IE browser.

Commit:
Fix impasse_custom_fields_tabs bug

Commit:
Merge branch 'master' into develop

Commit:
Merge branch 'fix_ie_no_version_2' into develop

Commit:
Fix impasse_custom_fields_tabs nil

Commit:
Fix impasse_custom_fields_tabs nil

cbeckr committed:
Made expected_results optional

cbeckr committed:
Merge branch 'master' into develop

Commit:
Added overview statistics

Commit:
Added czech translation

wschramm committed:
Update app/views/impasse_executions/index.html.erb

Adding fix for issue 172

Commit:
Added untranslated Test Coverage

Commit:
Selected version in coverage is highlighted.
All version are shown, including completed.

mindo committed:
some labelled_tabular_form_for labels were unchanged and the plugin was not working

cforce committed:
Create de.yml

fixes #173

mindo committed:
for some unknown reason commit dbca8424e2479b6c05eff74c12b1625d7e6a0e60 was not imported from original repo

cforce committed:
Merge branch 'develop' of https://github.com/gartz/redmine_impasse into develop

Conflicts:
	app/controllers/impasse_execution_bugs_controller.rb
	app/views/impasse_execution_bugs/_form.html.erb

cforce committed:
Merge branch 'develop' of https://github.com/mes/redmine_impasse into develop

cforce committed:
Merge branch 'master' of https://github.com/tvrzna/redmine_impasse into develop

Conflicts:
	app/models/impasse/test_plan.rb
	app/views/impasse_test_case/index.html.erb

cforce committed:
Merge branch 'patch-1' of https://github.com/cforce/redmine_impasse into develop

cforce committed:
Merge branch 'master' of https://github.com/thumbtack-technology/redmine_impasse into develop

Conflicts:
	app/views/impasse_test_case/index.html.erb

cforce committed:
Merge branch 'master' of https://github.com/mindo/redmine_impasse into develop

Conflicts:
	app/views/impasse_settings/_form.html.erb
	app/views/impasse_test_case/index.html.erb

cforce committed:
Merge branch 'master' of https://github.com/wschramm/redmine_impasse into develop

cforce committed:
Update init.rb

Commit:
Fixed database user, public user removed and fixed selectors that where searching for one value to multiple values

cforce committed:
Merge pull request #1 from gartz/develop

Fixed database user, public user removed and fixed selectors that where ...

Commit:
add lables for coverage

cforce committed:
Merge pull request #2 from mkinski/develop

add lables for coverage

cforce committed:
Merge branch 'develop' of https://github.com/cforce/redmine_impasse

cforce committed:
reload paged conetn in ajax request in div - fix #178 fix #186

fixes #178
fixes #186

made a "work around fix. I insert js script which binds the page clicks events and fires a ajax call to reload the div content with the paged content and overrdide the href so no onw view request is made.

cforce committed:
fixes #178
fixes #186

made a "work around fix. I insert js script which binds the page clicks events and fires a ajax call to reload the div content with the paged content and overrdide the href so no onw view request is made.

halimali commented:
cool.. working changes on redmine 2.3.2.
Thanks for integrating all the patches.

cforce committed:
fixes #178
fixes #186
fixes #187

ricardobauqa commented:
Hi guys any news about this pull request? 

Best Regards

Ricardo.

viklele commented:
Hi,

Is this project still active? 

Appears to be an excellent choice for test case management under Redmine, but the lack of updates and activity by project owner is making it a bit risky choice...especially because I am not familiar with Ruby / Rails programming.

Regards,
- Vikram

cforce committed:
Update impasse_test_case_controller.rb

Change accpording #221

cforce committed:
Update impasse_test_case_controller.rb

Change according #221

cforce committed:
Merge pull request #5 from cforce/patch-3

Bugfix: impasse creating/editing unsorted test cases

cforce committed:
Merge pull request #6 from cforce/patch-2

reload paged conetn in ajax request in div - fix #178 fix #186

cforce committed:
Create es.yml

https://github.com/Saragu0/redmine_impasse

guilhermenoronha committed:
Bug Fix:

- Re-added params "project" to functions "get_statistics_for_plan" and "get_statistics"
- Re-added params "project" where these functions are called.
- Added on query of these functions above the condition "AND project_id = project"

This small bug fix was necessary to correct two problems. They are:

- To view the statistics on the page "plans of tests" for old versions of redmine_impasse.
- Fix the error correlated with "plans of tests" with the same name, but in different projects. With this, we can copy plans of tests to different projects and keep them with the same name.

cforce committed:
Merge pull request #7 from guilhermenoronha/develop

Corrections to view projects in old versions of impasse and create plans of tests with the same name on different versions.

Tx for ur contribution
